commune_app/views.py

```diff
@@ -107,3 +126,29 @@ def vote(request):
 def user_logout(request):
     logout(request)
     return redirect('main_page')
+
+
+def create_commune(request):
+    if request.method == 'POST':
+        name = request.POST['name']
+        description = request.POST['description']
+        wallet = request.POST['wallet']
+        new_commune = Commune.create_commune(name=name, description=description, wallet=wallet)
+        new_commune.save()
+        user_id = request.user.id
+        User = get_user_model()
+        user = User.objects.get(id=user_id)
+        user.join_commune(new_commune.id)
+        return redirect('main_page')
+    else:
+        return render(request, 'commune_app/create_commune.html')
+
+
+def do_chore(request):
+    if request.method == 'POST':
+        chore_id = request.POST['chore_id']
+        chore = Chore.get_chore(chore_id)
+        chore.execute_chore(chore_id=chore_id, user_id=request.user.id)
+        return redirect('commune')
+    else:
+        return redirect('commune')
```

Comment:
if the return is the same for all options so you can do it once.
like:
```suggestion
    return redirect('commune')
```